scenes/game_elements/props/sequence_puzzle_hint_sign/components/sequence_puzzle_hint_sign.gd

```diff
@@ -136,8 +147,12 @@ func _set_solved_ambient_sound(new_sound: AudioStream) -> void:
 
 func _get_configuration_warnings() -> PackedStringArray:
 	var warnings: PackedStringArray
+	var required_animations: Array[StringName] = REQUIRED_ANIMATIONS.duplicate()
+
+	if not interact_demonstrates:
+		required_animations.append(&"hint")
```

Comment:
I find it a bit hard to read because "hint" is supposed to be optional but is added to a variable named "required". What about?

```suggestion
	var all_animations: Array[StringName] = REQUIRED_ANIMATIONS.duplicate()

	if not interact_demonstrates:
		all_animations.append(&"hint")
```